prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -133,3 +133,37 @@ resource "google_monitoring_alert_policy" "heartbeat-job-stale" {
   # gcloud beta monitoring channels list --project=oss-prow
   notification_channels = ["projects/${var.project}/notificationChannels/${var.notification_channel_id}"]
 }
+
+resource "google_monitoring_alert_policy" "probers" {
+  project = var.project
+
+  display_name = "HostDown"
+  combiner     = "OR"
+  conditions {
+    display_name = "Host is unreachable"
+    condition_monitoring_query_language {
+      duration = "120s"
+      query    = <<-EOT
+      fetch uptime_url
+      | metric 'monitoring.googleapis.com/uptime_check/check_passed'
+      | filter (metric.check_id == 'blackbox-exp')
```

Comment:
What is this field and where does its value come from? I only see this value in the existing metrics.

Reply:
this was from the experiment, removed